### DISCUSSION TITLE
Welcome to EduPage2 Discussions!

### DISCUSSION THREAD

**Discussion by vyPal** [DislikesSchool/EduPage2#54] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by orpa71** [comment]
This project is cool.

I didnt expect that there would be a third party client for a service that has pretty bad UI design, doesnt have modern features like dark mode, and more
However, its not perfect, there still are some missing features, like hiding, marking messages/homework as finished, and other things

Nonetheless i respect the effort you put into this, as its impressive (in my opinion) for a single person to make a fully fledged app like this. Thanks for creating this.
